_RELEASE/config.json

```diff
@@ -29,7 +29,7 @@
 	"no_music" : false,
 	"no_rotation" : false,
 	"no_sound" : false,
-	"official" : true,
+	"official" : false,

```

Comment:
Please set this back to true. It's annoying when a build is off official mode by default.